### DISCUSSION TITLE
Next/image not optimizing image sizes very well

### DISCUSSION THREAD

**Discussion by iojcde** [vercel/next.js#24472] [Help]
Hi, 
I am using next/image to convert images into smaller ones. The images in question are used as profile pictures. However, they're still pretty big, 256 width is too big for a 70 width profile area. Is there any way to make the image smaller(while using next/image) without shrinking them by hand?

Link: https://teamint.xyz
Repo: https://lab.jcde.xyz/team-int/web

![IMG_20210426_221223](https://user-images.githubusercontent.com/31413538/116088083-7e7bf180-a6dc-11eb-98c5-23022edb394b.jpg)

**Comment by bobarros** [comment]
It's hard to use the algorithm and then trying to change its decisions.

Are you trying with different DPR's? I changed to different ones and with 1.0 DPR the result dropped to 96w with Chrome. With Firefox, the result was 128w. 

I also realized that your original webp file has 512w and 25kb. The 256w one has only 10kb.

It seems fair to me.
